kafka-client/src/main/java/dev/responsive/kafka/internal/stores/RemoteWindowOperations.java

```diff
@@ -213,6 +232,26 @@ private static RemoteWindowTable<?> createMongo(
     }
   }
 
+  private static RemoteWindowTable<?> createRs3(
+      final ResponsiveWindowParams params,
+      final SessionClients sessionClients,
+      final ResponsiveConfig config,
+      final ResponsiveMetrics responsiveMetrics,
+      final ResponsiveMetrics.MetricScopeBuilder scopeBuilder
+  ) {
+    if (params.schemaType() == SchemaTypes.WindowSchema.STREAM || params.retainDuplicates()) {
+      throw new UnsupportedOperationException("Duplicate retention is not yet supported in RS3");
+    }
+
+    // TODO: Pass through retention period once we have support for TTL
```

Comment:
FYI I'm actually doing this as part of the [CreateStore implementation](https://github.com/responsivedev/responsive-pub/pull/440/files#diff-7093c37c9cd2cd8b4ee477e375bb6a7802c8e134cf1e540cfd7505119aaced47R282)

It's just setting the default ttl for the store (so we still need the STREAM_TIME clock type for full window store ttl support) but we can do the same thing here. So I can handle this as part of integrating the CreateStore API with the RS3 window stores after both PRs are merged